library/aiter-gfx908/build-aiter-gfx908.sh

```
@@ -0,0 +1,298 @@
+INCLUDE_FP8_GEMM="${INCLUDE_FP8_GEMM:-0}"
+# CK MHA/FMHA family: huge CK kernels that time out on gfx908; MI100 serves
+# attention via the CK-FA / Triton backends, not AITER MHA.  Skipped by
+# default; opt in with INCLUDE_MHA=1.
+INCLUDE_MHA="${INCLUDE_MHA:-0}"
+
+echo "==> AOT-building AITER modules for ${GFX_ARCH} (no runtime JIT at serve)"
+echo "    PER_MODULE_TIMEOUT=${PER_MODULE_TIMEOUT}s  INCLUDE_FP8_GEMM=${INCLUDE_FP8_GEMM}  INCLUDE_MHA=${INCLUDE_MHA}"
+MODULES="${MODULES:-}" FORCE_REBUILD="${FORCE_REBUILD}" \
+PER_MODULE_TIMEOUT="${PER_MODULE_TIMEOUT}" INCLUDE_FP8_GEMM="${INCLUDE_FP8_GEMM}" \
+INCLUDE_MHA="${INCLUDE_MHA}" \
+PYBIN="${PYBIN}" "${PYBIN}" - <<'PY'
```

Shellcheck (0.11.0)

[warning] This assignment is only seen by the forked process.

(SC2097)